doc/manual/src/contributing/hacking.md

```diff
@@ -83,7 +83,7 @@ by:
 $ nix develop
 ```
 
-## Testing
+## Testing Nix
```

Comment:
I think it should be clear that it's about Nix.

```suggestion
## Running tests
```

Reply:
The titles "Testing Nix" and "Testing the install scripts" aren't trying to clarify that you're here to test the Nix project, they're trying to clarify that the testing procedures for Nix itself and its installer scripts are separate.

Not married to the phrasing, but "Running tests" erases this distinction and suggests it runs when all of the tests when it doesn't?

Reply:
Ah, sorry, I misread the heading level to be different. Updated [the other suggestion](https://github.com/NixOS/nix/pull/6652/files/9b7bd2dd1fc83f6df449fce3967a95cb098ca4b2#r972703644) to make it part of the list of test procedures.

Reply:
adopted change locally, available when I push